dockerfile_parse/parser.py

```diff
@@ -91,6 +93,10 @@ def __init__(self, path=None, cache_content=False, env_replace=True):
 
         self.env_replace = env_replace
 
+        if parent_env:
+            logger.debug("Setting inherited parent image ENV vars: %s", parent_env)
+            self.envs(parent_env)
```

Comment:
@lcarva Isn't this already being used to modify the Dockerfile in place to apply the new labels?